src/bittytax/conv/parsers/coinbase.py

```diff
@@ -220,7 +237,27 @@ def _do_parse_coinbase(
                 fee_asset=currency,
                 wallet=WALLET,
             )
-    elif row_dict["Transaction Type"] in ("Sell", "Advanced Trade Sell"):
```

Comment:
Same question as above, why did you split handling of "Sell" and "Advanced Trade Sell"?